lib/datasets.go

```diff
@@ -52,7 +50,9 @@ func (m DatasetMethods) Name() string {
 func (m DatasetMethods) Attributes() map[string]AttributeSet {
 	return map[string]AttributeSet{
 		"componentstatus": {endpoint: AEComponentStatus, httpVerb: "POST"},
-		"get":             {endpoint: AEGet, httpVerb: "GET"},
+		"get":             {endpoint: AEGet, httpVerb: "POST"},
+		"getcsv":          {endpoint: denyRPC},
+		"getzip":          {endpoint: denyRPC},
```

Comment:
In person, we decided that for now that needs to be supported over rpc!

Main points:
- not sure that there are many people who would expect to both be running a qri node somewhere & expect all functionality to still work the same way over cli
- desktop would still be able to fetch over the api via the sugar endpoints
- it would be simple to add in if someone requests it later